src/main/resources/db/migration/V1_117__update_print_files_to_clob_procedure.sql

```diff
@@ -0,0 +1,168 @@
+-- DROP PROCEDURE juror_mod.printfiles_to_clob(int4);
+
+CREATE OR REPLACE PROCEDURE juror_mod.printfiles_to_clob(IN p_limit integer DEFAULT 1000000)
+ LANGUAGE plpgsql
+AS $procedure$
+DECLARE
+	v_form_type VARCHAR(6);
+	v_max_rec_len INTEGER;
```

Comment:
Consider renaming this to be rec_len. It will just be holding the record length for the given form type. It's not a max.